test/Constraints/diagnostics.swift

```diff
@@ -428,7 +428,7 @@ let _: (Int) -> (Int, Color) = { ($0, .Unknown("")) } // expected-error {{missin
 let _: Color = .Unknown("") // expected-error {{missing argument label 'description:' in call}} {{25-25=description: }}
 let _: Color = .Unknown // expected-error {{member 'Unknown' expects argument of type '(description: String)'}}
 let _: Color = .Unknown(42) // expected-error {{missing argument label 'description:' in call}}
-let _ : Color = .rainbow(42)  // expected-error {{argument passed to call that takes no arguments}}
+let _ : Color = .rainbow(42)  // expected-error {{argument passed to call that takes no arguments}} {{26-28=}}
```

Comment:
Sorry, but IMO, we should not remove users code like this. This might be just a mistype of *function name*.
Removing `()` is implemented in https://github.com/apple/swift/pull/9583 because of migration need and it probably reflects users intent.